ce/package.json

```diff
@@ -37,39 +37,42 @@
   "homepage": "https://github.com/Microsoft/vcpkg-ce#readme",
   "readme": "https://github.com/Microsoft/vcpkg-ce/blob/master/readme.md",
   "devDependencies": {
-    "@types/node": "14.14.22",
-    "@typescript-eslint/eslint-plugin": "^4.31.1",
-    "@typescript-eslint/parser": "^4.31.1",
-    "@types/micromatch": "4.0.1",
+    "@types/node": "17.0.15",
+    "@typescript-eslint/eslint-plugin": "5.10.2",
+    "@typescript-eslint/parser": "5.10.2",
+    "@types/micromatch": "4.0.2",
     "eslint-plugin-notice": "0.9.10",
-    "eslint": "^7.32.0",
-    "@types/semver": "~7.3.4",
+    "eslint": "8.8.0",
+    "@types/semver": "7.3.9",
     "@types/tar-stream": "~2.2.0",
-    "typescript": "4.4.3",
-    "shx": "0.3.3",
-    "chalk": "4.1.0",
-    "translate-strings": "1.0.5",
+    "typescript": "4.5.5",
+    "shx": "0.3.4",
+    "chalk": "4.1.2",
+    "translate-strings": "1.0.11",
     "@types/marked-terminal": "3.1.3",
     "@types/marked": "4.0.2",
-    "@types/cli-progress": "3.9.1"
+    "@types/cli-progress": "3.9.2",
+    "@types/mocha": "9.1.0",
+    "source-map-support": "0.5.21"
   },
   "dependencies": {
     "@snyk/nuget-semver": "1.3.0",
-    "vscode-uri": "3.0.2",
+    "vscode-uri": "3.0.3",
     "ee-ts": "2.0.0-rc.6",
-    "yaml": "2.0.0-8",
-    "semver": "~7.3.4",
+    "yaml": "2.0.0-10",
+    "semver": "7.3.5",
     "tar-stream": "~2.2.0",
-    "got": "11.8.2",
-    "sorted-btree": "1.4.0",
-    "sed-lite": "~0.8.2",
-    "unbzip2-stream": "~1.4.3",
+    "got": "11.8.3",
+    "sorted-btree": "1.6.0",
+    "sed-lite": "0.8.4",
+    "unbzip2-stream": "1.4.3",
     "micromatch": "4.0.4",
-    "chalk": "4.1.0",
+    "chalk": "4.1.2",
     "marked-terminal": "5.1.1",
     "marked": "4.0.12",
-    "cli-progress": "3.9.0",
-    "applicationinsights": "2.1.9",
-    "fast-glob": "3.2.7"
+    "cli-progress": "3.10.0",
+    "applicationinsights": "2.2.1",
+    "fast-glob": "3.2.11",
+    "tar": "6.1.11"
```

Comment:
Why add a dependency on `tar` -- we don't use that.  

If you're looking to fix the `tar` that gets pulled in by PNPM, you need to update that in rush.hson.

(which is where most of the scan warnings comes from btw)

Update these in `rush.json`, 
```
  "rushVersion": "5.61.4",
  "pnpmVersion": "6.29.1",
```
And then erase your local `conmmon/config/rush/pnpm-lock.yaml`  file.

And then run `rush purge` and `rush update`.

Reply:
>Why add a dependency on tar -- we don't use that.

Because the version selected without that dependency is too old. Will try your suggestion instead.

Reply:
>(which is where most of the scan warnings comes from btw)

I don't see anything naming "tar" in there. The CG alerts that started this investigation explicitly indicated that the bad version comes because tar is selected only as a dependency of tar-stream.